pages/about-us/donate/en.text

```diff
@@ -39,6 +39,14 @@ h1. Online donations
 
 <%= render :partial => 'inc/donate_forms' %>
 
+h1. Wire transfers (EUR only)
```

Comment:
I think we should say "Wire/Bank transfers (EUR only)" -- because many of the germans who have written in asking to donate, always say "bank transfer' instead of 'wire'.